pkg/cmd/config/view.go

```diff
@@ -0,0 +1,67 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+
+	"github.com/spf13/cobra"
+)
+
+// NewCmdConfigView returns a new (config) view command.
+func NewCmdConfigView(f util.Factory, o *ViewOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "view",
+		Short: "Print the gardenctl configuration",
+		RunE:  base.WrapRunE(o, f),
+	}
+
+	o.AddFlags(cmd.Flags())
+
+	return cmd
+}
+
+// Run executes the command
+func (o *ViewOptions) Run(f util.Factory) error {
+	m, err := f.Manager()
+	if err != nil {
+		return fmt.Errorf("failed to get target manager: %w", err)
+	}
+
+	config := m.Configuration()
+	if config == nil {
+		return errors.New("could not get configuration")
+	}
+
+	return o.PrintObject(config)
+}
+
+// ViewOptions is a struct to support view command
+type ViewOptions struct {
```

Comment:
Why is this exported?

Reply:
It is only used in the tests. If you want, I can make it internal and move the tests to the package (make it whitebox tests)

Reply:
`export_test.go`